neqo-transport/src/recovery.rs

```diff
@@ -590,36 +591,55 @@ impl PtoState {
     }
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub(crate) struct LossRecovery {
     /// When the handshake was confirmed, if it has been.
     confirmed_time: Option<Instant>,
     pto_state: Option<PtoState>,
     rtt_vals: RttVals,
-    cc: CongestionControl,
+    packet_sender: PacketSender,
 
     spaces: LossRecoverySpaces,
 
     qlog: NeqoQlog,
     stats: StatsCell,
 }
 
+impl Default for LossRecovery {
+    fn default() -> Self {
+        Self {
+            confirmed_time: None,
+            pto_state: None,
+            rtt_vals: RttVals::default(),
+            packet_sender: PacketSender::new(CongestionControlAlgorithm::default()),
+            spaces: LossRecoverySpaces::default(),
+            qlog: NeqoQlog::default(),
+            stats: StatsCell::default(),
+        }
+    }
+}
```

Comment:
As I said previously, I don't think that we need a Default implementation for LossRecovery, or CongestionAlgorithm.  The latter is part of the constructor to Connection and we should be able to make a decision.  All of those `::default()` calls can be replaced by `::NewReno`.

Reply:
This default is used only by tests. I can remove the default.

Reply:
To be precise the default for LossRecovery.

Reply:
I made CongestionAlgorithm implements Default to make it easier to update to a new cc without changing cc everywhere.